README.md

```diff
@@ -1 +1,232 @@
-# JavaScript-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Microverse README Template</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 JS Capstone Module 2](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [JS Capstone Module 2] <a name="about-project"></a>
+
+> Its a daily rountine list for users to write down their task.
+
+**[JS Capstone]** is a project for microvesre.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Client
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[HTML]**
+- **[JavaScript]**
+- **[WebPack]**
+- **[Gitflow]**
+- **[GoogleApi]**
+- **[TestFunctions]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ [https://javed-shahbaz.github.io/JavaScript-Capstone](https://javed-shahbaz.github.io/JavaScript-Capstone/)
+
```

Comment:
- [ ] Dear @Javed-Shahbaz, Please remember we are not expected to have any broken links in our README file. As it's not mandatory to deploy for this project, Please make sure to remove the live link if you have not deployed the app yet. 👍🏻

Reply:
well resolved! ✔️